src/components/curvedText/curvedText.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+// Sourced from this gist: https://gist.github.com/garth/0ca97a1112cb969cb72d951a85d3a0fe
+/*
+Draws text around a circule (the object).
+Text is centered at the top of the circle.
+Depends on styled-components, but can be adapted for other frameworks.
+<CurvedText
+  text="I'm curved"
+  objectSize={100} // diameter of the circle to wrap the text around
+  spacing={12} // padding between the circle and text
+  offset={30} // ammount of space for text, make this bigger to stop larger text from being cropped
+  overlap={true} // sets the bottom margin to negative so that the text is centered around the object
+/>
+*/
+
+const CurvedText = (props) => {
+  const {
+    text,
+    fontSize,
+    objectSize,
+    spacing,
+    offset,
+    overlap
+  } = props
+
+  const d = objectSize + spacing * 2
+  const r = objectSize / 2 + spacing / 2
+  const Container = styled.div`
+    margin-bottom: ${overlap ? `-${r}px` : '0'};
+    height: ${r + offset}px;
+    color: ${props => props.theme.colors.navy};
+    
+    path {
+      fill: transparent;
+    }
+    text {
+      fill: currentColor;
+      text-anchor: middle;
+      font-size: ${fontSize};
```

Comment:
Should we set the `font-family` to be the same as the one we use for header elements? (H1, H2, etc)

Reply:
Good call!